src/navigators/app-navigator.tsx

```diff
@@ -24,6 +24,7 @@ import {
 } from "screens";
 import { colors } from "theme";
 import { SUGGESTED_USER_DATA_I } from "constant";
+import { RootState, useAppSelector } from "../store/store";
```

Comment:
Use absolute path for imports. I dont want to se .../.../../.. etc.
It should be store/store something